### ISSUE TITLE
door needs switch

### ISSUE THREAD

**Issue by x3003** [Realm667/WolfenDoom#1496]
In c3m6_b, this door needs a switch.

![Screenshot_Doom_20241028_234148](https://github.com/user-attachments/assets/b9e9c00f-73b2-49b6-9cd6-4710e7285bdf)

If you're in the castle, go to right door and get all the keys, you might wanna goto the 6 keys skull fast. But if you haven't opened this door before, you can't get through because of a missing switch.

**Comment by hawkwind3** [comment]
That's not the only one. There are other doors like this that can only be opened from one side. I'm guessing deliberate mapping design.

**Comment by x3003** [comment]
Yes, but this is the only one who prevents you from getting to the skull fast . Now you have to go all the way around the place to get to the skull (or clip).

**Comment by hawkwind3** [comment]
At least it doesn't cause a softlock. Might be authors' intent. If the author wishes to address this ...

**Comment by Talon1024** [comment]
As I understand it, C3M6_B (Castle Wolfenstein interior) is supposed to be open to explore **before** the final boss fight, and have only one route open to the "finish line" **after** the boss fight.

**Comment by hawkwind3** [comment]
What you say here might be related to #1494

**Comment by x3003** [comment]
> is supposed to be open to explore before the final boss fight, 

Yes, but normally you open / explore the left **and** right door inside the castle, If you choose not to go through the left door, you can but stuck at this door, [later] when you wanna make a quick route, out left, in right to get to the skull switch. Else you have to take the long route.

>What you say here might be related

No.

**Comment by Tormentor667** [comment]
The question is: Is this game-breaking? Talon already made the point here.

**Comment by x3003** [comment]
Game breaking, no, But personally, i'm not taking the long way round if it's closed.

**Comment by Ozymandias81** [comment]
This has been a mapping choice by DoomJuan to give players a chance of replayability, since the castle has tons of secrets and shortcuts on purpose.
